src/Data/Core.Persistence/DatabaseProvider.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace QuantumCore;
+
+public enum DatabaseProvider
```

Comment:
A possible default of `Unknown` or `Invalid` set to `0` would help detect miss-configuration problems I believe.

Reply:
AFAIK it throws an exception if the enum cannot be parsed (value/name not defined) so it validates it implicitly. Feel free to prove me wrong 👍